src/Api.Rest/Api.Rest.csproj

```diff
@@ -48,6 +48,7 @@
   </ItemGroup>
 
   <ItemGroup Label="NuGet references">
+    <PackageReference Include="CacheCow.Client" Version="2.8.3" />
```

Comment:
Additionally to this line we need to add this dependency to the build.cake (line 236) in the main directory so it is correctly included during build.

Reply:
done,
thanks